com.unity.ml-agents/CHANGELOG.md

```diff
@@ -39,6 +39,8 @@ sizes and will need to be retrained. (#5181)
 - Make com.unity.modules.physics and com.unity.modules.physics2d optional dependencies. (#5112)
 - The default `InferenceDevice` is now `InferenceDevice.Default`, which is equivalent to `InferenceDevice.Burst`. If you
 depend on the previous behavior, you can explicitly set the Agent's `InferenceDevice` to `InferenceDevice.CPU`. (#5175)
+ - Added support for `Goal Signal` as a type of observation. Trainers can now use HyperNetowrks to process `Goal Signal`. (#5142, #5159, #5149)
```

Comment:
Should we add a very short blurb as to what benefit using HyperNetworks does? e.g. "Trainers with hypernetworks are more effective at solving multiple tasks."